features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/DiscardOldEventsScenario.kt

```diff
@@ -52,6 +52,8 @@ internal class DiscardOldEventsScenario(
         Bugsnag.notify(MyThrowable("DiscardOldEventsScenario"))
 
         waitForEventFile()
+        // Extra sleep needed on Android 10+
+        Thread.sleep(2000)
```

Comment:
```suggestion
        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.Q) {
          // Extra sleep needed on Android 10+
          Thread.sleep(2000)
        }
```

Assuming we want to keep this sleep - maybe make it specific to the versions with the flake?

Reply:
I've made the comment a TODO to investigate, as I don't really want it in there at all (not without understanding it properly, anyway).